websites/M/MyTF1/presence.ts

```diff
@@ -0,0 +1,169 @@
+const presence = new Presence({
+		clientId: "1043951049800695808",
+	}),
+	strings = presence.getStrings({
+		viewHome: "general.viewHome",
+		viewPage: "general.viewPage",
+		viewProfile: "general.viewProfile",
+		watchingVid: "general.watchingVid",
+		browse: "general.browsing",
+		live: "general.live",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/k8jIoSZ.png",
+			startTimestamp: browsingTimestamp,
+		},
+		path = document.location.pathname,
+		time = await presence.getSetting<boolean>("time");
+
+	if (!time) delete presenceData.startTimestamp;
+
+	const { browse, live } = await strings;
+
+	if (path === "/") {
+		presenceData.details = (await strings).viewHome;
+		presenceData.state = browse;
+		delete presenceData.smallImageKey;
+		delete presenceData.smallImageText;
+		presenceData.largeImageKey = "https://i.imgur.com/k8jIoSZ.png";
+	} else if (path.startsWith("/tf1/direct")) {
+		presenceData.details = document.querySelector(
+			".VideoSummary_programName_xXfYS"
+		).textContent;
+		presenceData.state = document.querySelector(
+			".VideoSummary_title_o8ZzQ"
+		).textContent;
+		presenceData.smallImageKey = "https://i.imgur.com/xuvR04J.png";
+		presenceData.smallImageText = live;
+		if (
+			document.querySelector<HTMLImageElement>(
+				"[class*=LiveShowCardsSlider_list__card_uOvlu] picture img"
+			).src
+		) {
+			presenceData.largeImageKey = document
+				.querySelectorAll(".LiveShowCardsSlider_list__card_uOvlu")[0]
+				.querySelector("picture")
+				.querySelector("img").src;
```

Comment:
I have to use `.querySelectorAll(...)[0]` because I need to select a list item that matches the current program

![image](https://github.com/PreMiD/Presences/assets/70274512/dd53ff16-5343-4f51-944a-8541643017f6)